zig-dxread/src/main.zig

```diff
@@ -0,0 +1,72 @@
+const std = @import("std");
+const fs = std.fs;
```

Comment:
This is subjective but I usually just type `std.fs`

Reply:
i like that better thanks